### PR TITLE
refact(build): make the docker images configurable

### PR DESCRIPTION
**Pull request by kmova**
This commit will help customizing docker images via environment variables:
- The following ENV help specify custom docker build args
  - DBUILD_SITE_URL ( default: https://openebs.io )
  - DBUILD_REPO_URL ( default: https://github.com/openebs/jiva )
- The following ENV allows Travis to specify custom image org
  - IMAGE_ORG (default: openebs )
- The following ENV will allow specifying an extra tag suffix
  - TAG_SUFFIX ( default: none )

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

File: package/Dockerfile_build_amd64 [hunk 1]
```diff
@@ -4,5 +4,17 @@ RUN apt-get update && apt-get install -y curl \
 
 COPY longhorn jivactl launch copy-binary launch-with-vm-backing-file launch-simple-jiva /usr/local/bin/
 
+ARG ARCH
+ARG DBUILD_DATE
+ARG DBUILD_REPO_URL
+ARG DBUILD_SITE_URL
+
+LABEL org.label-schema.schema-version="1.0"
```
Comment (utkarshmani1997) [inline package/Dockerfile_build_amd64:12]: nit: should we also make version configurable?

File: package/Dockerfile_build_arm64 [hunk 2]
```diff
@@ -9,5 +9,17 @@ RUN apt-get update && apt-get install -y curl \
 
 COPY longhorn jivactl launch copy-binary launch-with-vm-backing-file launch-simple-jiva /usr/local/bin/
 
+ARG ARCH
+ARG DBUILD_DATE
+ARG DBUILD_REPO_URL
+ARG DBUILD_SITE_URL
+
+LABEL org.label-schema.schema-version="1.0"
```
Comment (utkarshmani1997) [inline package/Dockerfile_build_arm64:17]: same comment as above

File: package/Dockerfile_build_ppc64le [hunk 3]
```diff
@@ -9,5 +9,17 @@ RUN apt-get update && apt-get install -y curl \
 
 COPY longhorn jivactl launch copy-binary launch-with-vm-backing-file launch-simple-jiva /usr/local/bin/
 
+ARG ARCH
+ARG DBUILD_DATE
+ARG DBUILD_REPO_URL
+ARG DBUILD_SITE_URL
+
+LABEL org.label-schema.schema-version="1.0"
```
Comment (utkarshmani1997) [inline package/Dockerfile_build_ppc64le:17]: same comment as above

File: package/Dockerfile_build_ppc64le [hunk 4]
```diff
@@ -9,5 +9,17 @@ RUN apt-get update && apt-get install -y curl \
...
+LABEL org.label-schema.name="jiva"
+LABEL org.label-schema.description="OpenEBS Jiva"
+LABEL org.label-schema.build-date=$DBUILD_DATE
+LABEL org.label-schema.vcs-url=$DBUILD_REPO_URL
+LABEL org.label-schema.url=$DBUILD_SITE_URL
+
 VOLUME /usr/local/bin
 CMD ["longhorn"]
```
Comment (utkarshmani1997) [inline package/Dockerfile_build_ppc64le:25]: Now we are going to have enterprise version, should we change longhorn to jivactl?

File: package/Dockerfile_build_amd64 (hunk 1 above)
Comment (kmova) [reply package/Dockerfile_build_amd64:12]: this is the schema version, not the product version.

File: package/Dockerfile_build_ppc64le (hunk 4 above)
Comment (kmova) [reply package/Dockerfile_build_ppc64le:25]: we should probably take this up as its own. cc: @vishnuitta @payes what do you think? Make sure there are no breaking changes during upgrade and so forth.

**Pull request merged**